druid/src/menu.rs

```diff
@@ -566,8 +566,8 @@ pub mod sys {
             /// The 'Save' menu item.
```

Comment:
Update the comment here for Windows. Just take the macOS comment I think.

Reply:
Thanks! I knew I would forget something when splitting things up :sweat_smile: